vllm_gaudi/distributed/kv_transfer/kv_connector/v1/hpu_nixl_connector.py

```diff
@@ -3,13 +3,9 @@
 import math
 import threading
 import torch
-from vllm_gaudi.extension.logger import logger as init_logger
-from vllm.distributed.kv_transfer.kv_connector.v1 import (nixl_connector)
 from vllm.distributed.kv_transfer.kv_connector.v1.nixl_connector import (NixlAgentMetadata, NixlConnectorWorker)
-
-logger = init_logger()
-
-nixl_connector._NIXL_SUPPORTED_DEVICE = {"cuda": ("cuda", ), "tpu": ("cpu", ), "hpu": ("cpu", )}
```

Comment:
Can we support hpu device for host and device memory and use kv-transfer-config flags to switch devices.

nixl_connector._NIXL_SUPPORTED_DEVICE = {"cuda": ("cuda", ), "tpu": ("cpu", ), "hpu": ("cpu", "hpu")}
--kv-transfer-config '{\"kv_connector\":\"NixlConnector\",\"kv_role\":\"kv_both\",\"kv_buffer_device\":\"hpu\"}'"